### ISSUE TITLE
Incorrect partials syntax in posts/single.html

### ISSUE THREAD

**Issue by j7an** [luizdepra/hugo-coder#187]
Partials in single.html are not being processed because they're missing the `.html`

The syntax based on [Hugo documentation](https://gohugo.io/templates/partials/) is `{{ partial "<PATH>/<PARTIAL>.html" . }}`

tags & categories
```
{{ with .Page.Params.Categories }}{{ partial "taxonomy/categories" . }}{{ end }}
{{ with .Page.Params.Tags }}{{ partial "taxonomy/tags" . }}{{ end }}
```
series & disqus
```
{{ partial "posts/series" . }}
{{ partial "posts/disqus" . }}
```
math
```
{{ partial "posts/math" . }}
```

**Comment by luizdepra** [comment]
Which version of Hugo are you using?
I'm using the `v0.55.6/extended` and everything is working fine.
Have you tested this approach?

But, well, we could fix it anyway. Could you open a PR fixing the issue?

**Comment by j7an** [comment]
I'm using the latest which is `v0.55.6` installed via `brew`. Looks like brew install extended as displayed on line 22 via their install script [here](https://github.com/Homebrew/homebrew-core/blob/master/Formula/hugo.rb)

Yes, I'll get a PR going soon. Thanks.

**Comment by luizdepra** [comment]
Ok.  Closed in #188.
